docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -703,6 +704,36 @@ The ability to create a project for an earlier TFM depends on having that versio
 
 ***
 
+## `api`
+
+- **`--aot`**
+
+  Enable publish using AOT. For more information, see [ASP.NET Core support for native AOT](/aspnet/core/fundamentals/native-aot).
```

Comment:
Should you link here instead to stay in .NET docs?

https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/

Reply:
I agree, let's try to keep this in .NET docs.
```suggestion
  Enable publish using AOT. For more information, see [.NET support for native AOT](/dotnet/core/deploying/native-aot).
```

Reply:
The api template is an ASP.NET template, and the article I link to has a whole section devoted to that template.